apps/full-site-editing/full-site-editing-plugin/editor-site-launch/src/launch-menu/index.tsx

```diff
@@ -16,8 +16,8 @@ import './styles.scss';
 
 const LaunchStepMenuItemTitles = {
 	[ LaunchStep.Name ]: __( 'Name your site', 'full-site-editing' ),
-	[ LaunchStep.Domain ]: __( 'Select a domain', 'full-site-editing' ),
-	[ LaunchStep.Plan ]: __( 'Select a plan', 'full-site-editing' ),
+	[ LaunchStep.Domain ]: __( 'Choose a domain', 'full-site-editing' ),
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Choose a domain' )` *ES Score: 7*